src/plugins/content_management/server/event_stream/es/types.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Client } from '@elastic/elasticsearch';
+
+export type EsClient = Omit<
+  Client,
+  'connectionPool' | 'serializer' | 'extend' | 'close' | 'diagnostic'
+>;
+
+/**
+ * Represents a single event as it is stored in Elasticsearch.
+ */
+export interface EsEventStreamEventDto {
```

Comment:
Not sure what the `Dto` suffix means 😊

Reply:
It is like POJO, but DTO ([Data Transfer Object](https://en.wikipedia.org/wiki/Data_transfer_object)). It is like a plain JSON object which you intend to transfer between processes, e.g. because you store it in DB (our use case), or disk, or sending over network.

Reply:
From Wikipedia:

> ... DTO does not have any behavior except for storage, retrieval, serialization and deserialization of its own data. In other words, DTOs are simple objects that should not contain any business logic but may contain serialization and deserialization mechanisms for transferring data over the wire.